routes/register.go

```diff
@@ -398,6 +398,12 @@ func Register(c Config) {
 							r.Get("/{snapshot_id}/lay_summaries/{lay_summary_id}/confirm-delete", publicationediting.ConfirmDeleteLaySummary).Name("publication_confirm_delete_lay_summary")
 							r.Delete("/lay_summaries/{lay_summary_id}", publicationediting.DeleteLaySummary).Name("publication_delete_lay_summary")
 
+							// datasets
+							r.Get("/datasets/add", publicationediting.AddDataset).Name("publication_add_dataset")
+							r.Get("/datasets/suggestions", publicationediting.SuggestDatasets).Name("publication_suggest_datasets")
+							r.Delete("/datasets/{dataset_id}", publicationediting.DeleteDataset).Name("publication_delete_dataset")
+							r.Post("/datasets", publicationediting.CreateDataset).Name("publication_create_dataset")
+
```

Comment:
There's already a dataset routes block around line 422. I'm trying to keep these organised in the same order they appear in the views.

Reply:
@verheyenkoen I'll move them after merge